### PR TITLE
Mosviz with ipyvuetify

### PR DESCRIPTION
**Pull request by javerbukh**
Currently implemented: Back button, Next button

Present but not working: Drop down (in vuetify terms it is an OverflowBtn)

Not yet vuetified: ipysheet (mosviz table), Image viewer, 2D viewer, 1D viewer,  text box

Blocker: OverflowBtn does not allow events. I am opening an issue for this in the ipyvuetify repo

### CODE REVIEW

**Comment by javerbukh** [comment]
Everything is vuetified, however I think we need to solidify where the example datasets will live so that the examples/ directory does not need to change as often.

**Comment by ibusko** [comment]
Apparently I have no permissions to acces the repo. I tried to get it as a tar ball, buth then I get a permission problem when building.

**Comment by astrofrog** [comment]
Just a quick note - ipysheet needs to be included in the list of dependencies

File: examples/mosviz-example.ipynb [hunk 1]
```diff
@@ -2,12 +2,25 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
Comment (astrofrog) [inline examples/mosviz-example.ipynb]: Make sure you clear the output from the cells before committing the notebook (maybe just add a new commit with the output clearerd)

**Comment by javerbukh** [comment]
@astrofrog Made the requested changes. I'm not sure how to proceed about the merge conflicts at this time. If it's ok with @brechmos-stsci and @SaOgaz (and if this commit works), can you show this PR for the mosviz demo tomorrow and I will work on resolving the merge conflict next week when I return from vacation?

**Pull request merged**